### PR TITLE
Bp 476 entity mapping deletion

### PR DESCRIPTION
**Pull request by tommartensen**

### CODE REVIEW

File: src/SettingsView/EventType.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ class EventType extends ConnectionComponent {
 	
 	render() {
```
Comment (hpibptlab) [inline src/SettingsView/EventType.js]: MAJOR This function has 101 lines, which is greater than the 100 lines authorized. Split it into smaller functions. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS138)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: src/SettingsView/EventType.js [hunk 2]
```diff
@@ -166,5 +167,19 @@ class EventType extends ConnectionComponent {
 				}
 			})
 		}
-	})
+	}),
+    deleteMapping: mapping => ({
+        deleteMappingResponse: {
+            url: config.backendRESTRoute + `/entitymapping/${mapping.Id}/delete`,
+            method: 'DELETE',
+            body: "",
```
Comment (julianweise) [inline src/SettingsView/EventType.js]: delete body, it is not necessary here

**Review by julianweise — CHANGES_REQUESTED** [review]
Good job!

**Review by julianweise — APPROVED** [review]

**Pull request merged**